template/project.md

```diff
@@ -0,0 +1,13 @@
+# Template for adding a project to realworld.md
+
+Please add your project in alphabetical order among the existing projects and save your image in the `/img/` [directory](https://github.com/mfem/web/tree/master/src/img).
+
+## Project Name
+
+<img src="../img/placeholder.png" align="left" style="margin-top: 2px; margin-right: 10px" width="225" alt="placeholder image">
+
+Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem. Ut enim ad minima veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur.
```

Comment:
I like how you used the actual Cicero here rather than the usual mangled Lorem Ipsum!  I believe that last sentence should end in a question mark, and I recommend finishing out the passage for all of the Cicero lovers out there:
Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam nihil molestiae consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur?